test_runner/fixtures/neon_fixtures.py

```diff
@@ -453,6 +453,7 @@ def __init__(
         pageserver_get_vectored_concurrent_io: str | None = None,
         pageserver_tracing_config: PageserverTracingConfig | None = None,
         pageserver_import_config: PageserverImportConfig | None = None,
+        disable_kick_secondary_downloads: bool = False,
```

Comment:
something like `storcon_disable_kick_secondary_downloads` would make it a bit more obvious that this is storage controller related -- most of the other items here have a safekeeper_ or pageserver_prefix.